readme.md

```diff
@@ -46,8 +46,26 @@ To use a custom SVG, provide its url in the `src` attribute to request the exter
 <ion-icon src="/path/to/external/file.svg"></ion-icon>
 ```
 
+### Custom Asset Path
+
+If you have a different set of icons you like to load or if the Ionicon icons are hosted on a different page or path, you can set the asset url from which Ionicons pulls the icons via:
+
+```ts
+import { setAssetPath } from '@stencil/core'
+
+// set root path for loading icons to "<root>/public/svg"
+setAssetPath(`${window.location.origin}/public/svg/`);
+```
+
+This allows the use of named icons like this:
+
+```html
+<!-- now pulls the svg from "<root>/public/svg/heart.svg" -->
+<ion-icon name="heart"></ion-icon>
+```
+
 ## Variants
-Each app icon in Ionicons has a `filled`, `outline` and `sharp` variant. These different variants are provided to make your app feel native to a variety of platforms. The filled variant uses the default name without a suffix. Note: Logo icons do not have outline or sharp variants.
+Each app icon in Ionicons has a `filled`, `outline` and `sharp` variant. These different variants are provided to make your app feel native to a variety of platforms. The filled variant uses the default name without a suffix. Note: Logo icons do not have outlines or sharp variants.
```

Comment:
```suggestion
Each app icon in Ionicons has a `filled`, `outline` and `sharp` variant. These different variants are provided to make your app feel native to a variety of platforms. The filled variant uses the default name without a suffix. Note: Logo icons do not have outline or sharp variants.
```

The original wording here is correct. We use the "outline" term to refer to the icon variant (filled, outline, or sharp)